Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -39,22 +39,23 @@ using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_.
 .. code:: bash
 
     export CLUSTER_NAME=cluster1
-    gcloud container clusters create $CLUSTER_NAME --image-type COS --num-nodes 2 --machine-type n1-standard-4
+    export CLUSTER_ZONE=us-west2-a
```

Comment:
Should we mention the list of available regions here somewhere? https://cloud.google.com/compute/docs/regions-zones?hl=en_US#available

Reply:
I added a note about the `--zone`/`--region` flag along with a link to available locations.